api/src/scripts/populate_db.py

```diff
@@ -123,39 +125,47 @@ def add_entity(entity, priority):
         if self.df is None:
             return
 
-        # Keep a dict (map) of stable_id -> feed so we can reference the feeds when processing the static_reference
+        # Gather all stable IDs
+        stable_ids = [f"mdb-{int(row['mdb_source_id'])}" for index, row in self.df.iterrows()]
+
+        # Query once to get all feeds
+        gtfs_feeds = self.db.session.query(Gtfsfeed).filter(Gtfsfeed.stable_id.in_(stable_ids)).all()
+        gtfs_rt_feeds = self.db.session.query(Gtfsrealtimefeed).filter(Gtfsrealtimefeed.stable_id.in_(stable_ids)).all()
+
+        # Keep a dict (map) of stable_id -> feed, so we can reference the feeds when processing the static_reference
         # and the redirects.
-        feed_map = {}
+        feed_map = {feed.stable_id: feed for feed in gtfs_feeds + gtfs_rt_feeds}
+
         for index, row in self.df.iterrows():
             mdb_id = f"mdb-{int(row['mdb_source_id'])}"
             self.logger.debug(f"Populating Database for with Feed [stable_id = {mdb_id}]")
 
             # Feed
             feed_class = Gtfsfeed if row["data_type"] == "gtfs" else Gtfsrealtimefeed
-            feed = feed_class(
+            feed = feed_map[mdb_id] if mdb_id in feed_map else feed_class(
                 id=generate_unique_id(),
-                data_type=row["data_type"],
-                feed_name=row["name"],
-                note=row["note"],
-                producer_url=row["urls.direct_download"],
-                authentication_type=str(int(row.get("urls.authentication_type", "0") or "0")),
-                authentication_info_url=row["urls.authentication_info"],
-                api_key_parameter_name=row["urls.api_key_parameter_name"],
-                license_url=row["urls.license"],
                 stable_id=mdb_id,
-                status=row["status"],
-                provider=row["provider"],
             )
-
             feed_map[mdb_id] = feed
+            feed.data_type = row["data_type"]
```

Comment:
Here I would put:
    # Initialize the feed data. If the feed was already in the DB replace its data.

Reply:
I don't think this comment is necessary. I think the code is clear enough. -- Theses lines of code are to update the feed information wether it's new or not to make it consistent with the csv.